packages/web/utils/meTokens.tsx

```diff
@@ -0,0 +1,204 @@
+import { Orbis } from '@orbisclub/orbis-sdk';
+import { CONFIG } from 'config';
+import erc20Abi from 'contracts/erc20.abi';
+import FoundryFacetAbi from 'contracts/FoundryFacet.abi';
+import HubFacetAbi from 'contracts/HubFacet.abi';
+import MeTokensRegistryABI from 'contracts/meTokensRegistry.abi';
+import type { Signer } from 'ethers';
+import { BigNumber, Contract, ethers } from 'ethers';
+
+export const metokenRegistry = '0x8b91FcF2230ab04A46e2D83aaF062EC1B5AAAa5c';
+export const metokenFactory = '0x8D4ee3599aF814bF3Aa884c161f0dE81d9e97225';
+export const hubFacet = '0x4555cf6E984186F6C0dfeba1A26764b21553B39f';
+export const foundryFacet = '0xA56AAF637b057a5EDf7b7252D0B7280042E71335';
+export const metokenDiamond = '0x0B4ec400e8D10218D0869a5b0036eA4BCf92d905';
+export const nullMeToken = '0x0000000000000000000000000000000000000000';
+
+const COINGECKO_API_URL = 'https://api.coingecko.com/api/v3/coins/';
+const COINQUERY = '?localization=false&tickers=true&market_data=true';
+export const DAIADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F';
+
+export const getCollateralData = async (collateralTokenAddress: string) => {
+  let id;
+  switch (collateralTokenAddress) {
```

Comment:
what is the point of this switch statement?

Reply:
There will be more collateral tokens, we match them with their ID to query coingecko api